[Bug]: tuxpaint build failed for [arm]

### Problem description


...Compiling PostScript print support...

clang-14: warning: optimization flag '-ffloat-store' is not supported [-Wignored-optimization-argument]
...Preparing translation files...
msgfmt -o trans/ach.mo src/po/ach.po
msgfmt -o trans/af.mo src/po/af.po
clang-14: warning: optimization flag '-ffloat-store' is not supported [-Wignored-optimization-argument]
msgfmt: error while opening "trans/ach.mo" for writing: No such file or directory
src/fonts.c:1332:11: warning: comparison of array 'fi->filename' not equal to a null pointer is always true [-Wtautological-pointer-compare]
  if (fi->filename != NULL)
      ~~~~^~~~~~~~    ~~~~
msgfmt: error while opening "trans/af.mo" for writing: No such file or directory
make: *** [Makefile:524: trans/ach.mo] Error 1
make: *** Waiting for unfinished jobs....
make: *** [Makefile:524: trans/af.mo] Error 1
In file included from src/dirwalk.c:62:
In file included from src/fonts.h:43:

### What steps will reproduce the bug?

build

### What is the expected behavior?

_No response_

### System information

termux-info:

```

```

--- a/x11-packages/tuxpaint/build.sh
+++ b/x11-packages/tuxpaint/build.sh
@@ -36,3 +36,8 @@ termux_step_pre_configure() {
 
 	LDFLAGS+=" -landroid-wordexp"
 }
+
+termux_step_post_configure() {
+	# https://github.com/termux/termux-packages/issues/12458
+	mkdir -p trans
+}

